docs/api.rst

```diff
@@ -11713,7 +11716,7 @@ Delete mappings in the addressbook
 
     :reqjson object entries: A list of addresses to be deleted from the addressbook
     :reqjson str address: The address that will be deleted in the addressbook.
-    :reqjson str[optional] blockchain: The chain for which to delete the name mapping. If is not provided the names for all the chains will be deleted.
+    :reqjson str[optional] blockchain: The blockchain for which to delete the name mapping. If is not provided the names for all the chains will be deleted.
```

Comment:
Change does not matter.
But chain/blockchain is the same thing for us. Only if it said EVM chain you needed to touch this.